src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -528,21 +528,16 @@ RequestContext<object> requestContext
         {
             if (completionItem.Kind == CompletionItemKind.Function)
             {
-                RunspaceHandle runspaceHandle =
-                    await editorSession.PowerShellContext.GetRunspaceHandle();
-
                 // Get the documentation for the function
                 CommandInfo commandInfo =
-                    CommandHelpers.GetCommandInfo(
+                    await CommandHelpers.GetCommandInfo(
                         completionItem.Label,
-                        runspaceHandle.Runspace);
+                        this.editorSession.PowerShellContext);
 
                 completionItem.Documentation =
-                    CommandHelpers.GetCommandSynopsis(
+                    await CommandHelpers.GetCommandSynopsis(
                         commandInfo,
-                        runspaceHandle.Runspace);
-
-                runspaceHandle.Dispose();
+                        this.editorSession.PowerShellContext);
             }
```

Comment:
Haha, yep.  In this case I'm just trying to reuse existing logic in `PowerShellContext.ExecuteCommand` instead of using `PowerShell.Invoke` directly.  A bit of a perf hit to go through ExecuteCommand and async twice in a row, I'm thinking I might come up with my own async pipeline helper class at some point in the future.